site/installing.md

```diff
@@ -4,23 +4,22 @@ menu_order: 20
 ---
 
 
-Weave Scope consists of three parts: the probe, the app and the user interface. It can be deployed either as a standalone configuration, or if you don’t want to manage the administration yourself, you can sign up for Weave Scope as a service.
+Weave Scope consists of three parts: the probe, the app and the user interface. It can be deployed either as a standalone configuration, or if you don’t want to manage the administration yourself, you can sign up for Weave Cloud.
 
 The following topics are discussed:
 
  * [Installing Scope on Docker](#docker)
-   * [In Service Mode](#docker-service)
    * [Using Weave Net](#docker-weave)
    * [Using Docker Compose](#docker-compose)
-   * [Using Docker Compose in Service Mode](#docker-compose-sercice)
- * [Installing Scope on Kubernetes](#k8s)
+   * [Using Docker Compose in Weave Cloud](#docker-compose-sercice)
```

Comment:
That link goes nowhere. Not only is it misspelled (always was), but there is no corresponding anchor; instead we have two 'docker-compose' anchors.